Sources/Prototyper/Prototyper.swift

```diff
@@ -36,9 +35,9 @@ public struct PrototyperSettings {
 }
 
 
-/// <#Description#>
+/// This class contains the information which are shared a cross the different classes
 public class PrototyperState: ObservableObject {
```

Comment:
I would suggest that we can make `PrototyperState` internal as none of its properties as well as functions are exposed as public as far as I can see.